packages/kotti-ui/source/kotti-filters/types.ts

```diff
@@ -42,6 +57,11 @@ export namespace KottiFilters {
 			LESS_THAN_OR_EQUAL = 'LESS_THAN_OR_EQUAL',
 		}
 
+		export enum MultiEnum {
```

Comment:
how is this defined just now and did NOT error before!! 

I see the `Operation.Any` below already relied on `Operation.MultiEnum`

Reply:
`MultiEnum` was defined before (previously on line 31)
Due to the re-naming of Number to Integer, the order of the variables changed